src/components/calculator.js

```diff
@@ -1,44 +1,34 @@
-import React from 'react';
+import React, { useState } from 'react';
 import calculate from '../logic/calculate';
+import btnValues from './btnNames';
+import '../App.css';
 
-class Calculator extends React.Component {
-  constructor(props) {
-    super(props);
-    this.state = {};
-    this.handleClick = this.handleClick.bind(this);
-  }
+const Calculator = () => {
+  const [calcObject, setCalcObject] = useState({});
 
-  handleClick(e) {
-    this.setState((state) => calculate(state, e.target.textContent));
-  }
+  const handleClick = (e) => {
+    e.preventDefault();
+    try {
+      setCalcObject({ ...calcObject, ...calculate(calcObject, e.target.textContent) });
+    } catch (error) {
+      return calcObject;
+    }
+    return 0;
+  };
 
-  render() {
-    const { next, total } = this.state;
-    return (
-      <div className="container">
-        <div className="calc-display">{next || total || 0}</div>
-        <button type="button" onClick={this.handleClick} className="gray-like-btn">AC</button>
-        <button type="button" onClick={this.handleClick} className="gray-like-btn">+/-</button>
-        <button type="button" onClick={this.handleClick} className="gray-like-btn">%</button>
-        <button type="button" onClick={this.handleClick} className="orange-like-btn">÷</button>
-        <button type="button" onClick={this.handleClick} className="gray-like-btn">7</button>
-        <button type="button" onClick={this.handleClick} className="gray-like-btn">8</button>
-        <button type="button" onClick={this.handleClick} className="gray-like-btn">9</button>
-        <button type="button" onClick={this.handleClick} className="orange-like-btn">x</button>
-        <button type="button" onClick={this.handleClick} className="gray-like-btn">4</button>
-        <button type="button" onClick={this.handleClick} className="gray-like-btn">5</button>
-        <button type="button" onClick={this.handleClick} className="gray-like-btn">6</button>
-        <button type="button" onClick={this.handleClick} className="orange-like-btn">-</button>
-        <button type="button" onClick={this.handleClick} className="gray-like-btn">1</button>
-        <button type="button" onClick={this.handleClick} className="gray-like-btn">2</button>
-        <button type="button" onClick={this.handleClick} className="gray-like-btn">3</button>
-        <button type="button" onClick={this.handleClick} className="orange-like-btn">+</button>
-        <button type="button" onClick={this.handleClick} className="gray-like-btn btn-zero">0</button>
-        <button type="button" onClick={this.handleClick} className="gray-like-btn">.</button>
-        <button type="button" onClick={this.handleClick} className="orange-like-btn">=</button>
-      </div>
```

Comment:
- [ ] Please fix your design according to the project guideline, Your equal `=` sign is not at the right position
- Your design :point_down: 

![image](https://user-images.githubusercontent.com/75126481/149509913-fe11bfe0-5a33-4c33-b70f-91cc8210ed42.png)

- Project design :point_down: 

![image](https://user-images.githubusercontent.com/75126481/149510723-cc8d6112-860e-4602-a9ec-18606637d64c.png)